CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 
 ## 0.16.0-dev
 
+### Changed
```

Comment:
I wasn't sure which version to put these under, I assumed `-dev` as it's not a fix for anything and it's part of what will be in the next release.